Source/Core/Core/PowerPC/JitArm64/JitArm64Cache.cpp

```diff
@@ -60,7 +66,7 @@ void JitArm64BlockCache::WriteLinkBlock(const JitBlock::LinkData& source, const
 {
   const Common::ScopedJITPageWriteAndNoExecute enable_jit_page_writes;
   u8* location = source.exitPtrs;
-  ARM64XEmitter emit(location);
+  ARM64XEmitter emit(location, location + 12);
```

Comment:
Is there a reason the number 12 was chosen? If so a comment explaining it couldn't hurt

Reply:
The number of instructions here is always 3, as documented here: https://github.com/dolphin-emu/dolphin/blob/3dac4ec31cdf166419682d37c1c1f0f4e4e9c6d7/Source/Core/Core/PowerPC/JitArm64/JitArm64Cache.cpp#L26